x/auth/ante.go

```diff
@@ -306,10 +310,10 @@ func ensureSufficientMempoolFees(ctx sdk.Context, stdTx StdTx) sdk.Result {
 }
 
 func setGasMeter(simulate bool, ctx sdk.Context, stdTx StdTx) sdk.Context {
-	// set the gas meter
 	if simulate || ctx.BlockHeight() == 0 {
```

Comment:
This line could use an explanatory comment (in simulation or in the genesis block, we don't meter gas)

Reply:
Added a simple comment. Let me know if you want me to elaborate further.